src/test/java/io/strimzi/ValidatingWebhookTest.java

```diff
@@ -147,13 +148,14 @@ public void testEvictionOnPodWithExistingAnnotations() {
 
     @Test
     public void testWrongName() {
```

Comment:
Maybe you can rename the test as it now tests wrong label rather then wrong name?